contents/blog/application-vs-infrastructure-open-source.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2021-01-01
+title: Why you should probably build an application layer open source startup
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: tim-glaser
+categories: ["CEO diaries"]
+---
+
+Alt. title: "Application vs infrastructe open source startups"
+Outline:
+- Defining application layer
+  - Examples of application layer: GitLab, HashiCorp?, Sentry, Mattermost  
+- Defining Infrastructure layer
+  - "Essentially SaaS apps that you can self host"
+  - Examples of infrastructure layer: Docker, Gatsby, TimescaleDB 
```

Comment:
Personally, I'd have seen Gatsby as application (given it is an entire application in its own right - it gives you an entire website you can run)

Reply:
In my mind it's basically a framework and thus their only way of monetising is offering cloud services